### PR TITLE
Add GitHub issue templates

### PR DESCRIPTION
**Pull request by zach2good**

### CODE REVIEW

**Comment by zach2good** [comment]
It appears as though GH has chewwed up their own code, from their own issue template editor. I'll fix it later

**Comment by WinterSolstice8** [comment]
Am I seeing duplicate "bug report" and "feature request" files? I can't say I know how this issue template stuff works so maybe it has some inheritance or something? The bug report looks like a website template

File: .github/ISSUE_TEMPLATE/-hammer--feature-request.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+---
+
+<!-- Remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm the following points: -->
+<!-- (it should look like this: - [x] I have ...) -->
+- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
+- [ ] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated
+- [ ] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
+
+## Describe the feature that is missing
```
Comment (WinterSolstice8) [inline .github/ISSUE_TEMPLATE/-hammer--feature-request.md]: perhaps just `## Describe the feature`? I think the current wording may discourage people from making feature requests for new functionality that's not necessarily directly related to FFXI gameplay

**Comment by zach2good** [comment]
Driveby fixed, please squash when ready

**Review by claywar — APPROVED** [review]

**Pull request merged**